elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -294,7 +312,10 @@ def render(element_html, data):
         if answer_name in data['submitted_answers']:
             student_submission = [{
                 'inner_html': attempt['inner_html'],
-                'indent': ((attempt['indent'] or 0) * TAB_SIZE_PX) + INDENT_OFFSET
+                'indent': ((attempt['indent'] or 0) * TAB_SIZE_PX) + INDENT_OFFSET,
+                'badge_type': attempt['badge_type'] if (feedback_type == 'first-wrong') and 'badge_type' in attempt else '',
+                'icon': attempt['icon'] if (feedback_type == 'first-wrong') and ('icon' in attempt) else '',
+                'first-wrong': feedback_type == 'first-wrong' 
```

Comment:
Can we somehow have just one variable here instead of three new ones? Maybe actually construct the HTML for the badge right here, and then if there shouldn't be a badge, just use the empty string? Or some other way of making it cleaner?

Reply:
I had some redundant checks here so the updated code should be a bit cleaner. I could construct the HTML here but I feel like that would be more busy than keeping the HTML template in the mustache file and just setting an additional attribute here.